digid_eherkenning/saml2/eherkenning.py

```diff
@@ -32,6 +35,214 @@
+                ).url,
+                "binding": "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Artifact",
+            },
+            "attributeConsumingServices": [
+                {
+                    "index": eherkenning_config.eh_attribute_consuming_service_index,
+                    "serviceName": eherkenning_config.service_name,
+                    "serviceDescription": eherkenning_config.service_description,
+                    "requestedAttributes": [
+                        {
+                            "name": "urn:etoegang:DV:%(oin)s:services:%(index)s"
+                            % {
+                                "oin": eherkenning_config.oin,
+                                "index": eherkenning_config.eh_attribute_consuming_service_index,
+                            },
+                            "isRequired": False,
+                        }
+                    ],
+                    "language": "nl",
+                },
+                {
+                    "index": eherkenning_config.eidas_attribute_consuming_service_index,
+                    "serviceName": eherkenning_config.service_name + " (eIDAS)",
+                    "serviceDescription": eherkenning_config.service_description,
+                    "requestedAttributes": [
+                        {
+                            "name": "urn:etoegang:DV:%(oin)s:services:%(index)s"
+                            % {
+                                "oin": eherkenning_config.oin,
+                                "index": eherkenning_config.eidas_attribute_consuming_service_index,
+                            },
+                            "isRequired": False,
+                        }
+                    ],
+                    "language": "nl",
+                },
+            ],
+            "NameIDFormat": "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified",
+            "x509cert": open(cert_file, "r").read(),
+            "privateKey": open(key_file, "r").read(),
+            "privateKeyPassphrase": eherkenning_config.key_passphrase,
+        },
+    }
+
+    telephone = eherkenning_config.technical_contact_person_telephone
+    email = eherkenning_config.technical_contact_person_email
+    if telephone and email:
+        setting_dict["contactPerson"] = {
+            "technical": {"telephoneNumber": telephone, "emailAddress": email}
+        }
+
+    if eherkenning_config.organization_url and eherkenning_config.organization_name:
+        setting_dict["organization"] = {
+            "nl": {
+                "name": eherkenning_config.organization_name,
+                "displayname": eherkenning_config.organization_name,
+                "url": eherkenning_config.organization_url,
+            }
+        }
+
+    if eherkenning_config.no_eidas:
+        setting_dict["sp"]["attributeConsumingServices"] = setting_dict["sp"][
+            "attributeConsumingServices"
+        ][:-1]
```

Comment:
probably better to store this in a variable and then use `list.remove(...)` - now this is a magic number relying on the ordering of the services.